.gitignore

```diff
@@ -183,3 +183,4 @@ dist/
 
 # Misc
 statistics.html
+npm
```

Comment:
what is this from?

Reply:
i see it generated from running npm command. i have version 6.1.0.
contents:
c:\jssdk\extensions\applicationinsights-dependencies-js>type npm
Installing packages i
Installing packages i
Installing packages i
Installing packages i
Installing packages i